doc/Configuration.md

```diff
@@ -1149,6 +1149,39 @@ The packet action could be:
 }
 ```
 
+### BREAKOUT_CFG
+
+This table is introduced as part of Dynamic Port Breakout(DPB) feature.

```

Comment:
I think we should be more explicitly saying this table is auto-generated during the system init, and is also changed when we breakout the ports by CLI.